.gitignore

```diff
@@ -11,3 +11,4 @@ tags
 *#
 z_local.sbt
 .DS_Store
+tmp/
```

Comment:
Just FYI - the default folder for ScalaMeter test results is `target/benchmarks` since version `0.7`.